pkg/deploy/api/v1/conversion_test.go

```diff
@@ -13,23 +13,44 @@ import (
 )
 
 func TestTriggerRoundTrip(t *testing.T) {
-	p := DeploymentTriggerImageChangeParams{
-		From: kapiv1.ObjectReference{
-			Kind: "DockerImage",
```

Comment:
We don't support DockerImage references: https://github.com/openshift/origin/blob/0d4de0b1801e8c58a28c8bcd9d12b5d0c46de1cb/pkg/deploy/api/validation/validation.go#L261